tools/azure-pipelines/build-apache-repo.yml

```
@@ -47,6 +47,9 @@ variables:
   MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
   CACHE_KEY: maven | $(Agent.OS) | **/pom.xml, !**/target/**
   CACHE_FALLBACK_KEY: maven | $(Agent.OS)
+  TESTCONTAINER_CACHE_KEY: testcontainer-cache | $(Agent.OS) | **/pom.xml, !**/target/**
```

```suggestion
  TESTCONTAINER_CACHE_KEY: testcontainer-cache | $(Agent.OS)
```
The used images are usually unrelated to the poms, so I'm not sure whether using the poms as a key makes sense.